Formula/clickhouse-odbc.rb

```diff
@@ -0,0 +1,48 @@
+class ClickhouseOdbc < Formula
+  desc "ClickHouse ODBC driver"
+  homepage "https://clickhouse.yandex"
+  url "https://github.com/yandex/clickhouse-odbc.git",
+    :tag => "v1.0.0.20180903",
+    :revision => "17ecca3993e164d2d49360ed9be68becc11b7b05"
+  # head "https://github.com/yandex/clickhouse-odbc.git"
+
+  bottle do
+    # sha256 "256" => :high_sierra
+    # sha256 "256" => :sierra
+    # sha256 "256" => :el_capitan
+  end
+
+  # option "with-debug", "Install debugging version with advanced logs"
+
+  depends_on "cmake" => :build
+  depends_on "libiodbc" => :build
+  depends_on "libtool" => :build
+  depends_on "ninja" => :build
+  depends_on "openssl" => :build
+
+  needs :cxx14
+
+  def install
+    args = std_cmake_args
+    # args << "-DCMAKE_BUILD_TYPE=Debug" if build.with? "debug"
+    mkdir "build" do
+      system "cmake", "-G", "Ninja", "..", *args
+      system "ninja"
+      # system "ninja", "install" # installs too many trash
+      lib.install "driver/libclickhouseodbc.dylib"
+      lib.install "driver/libclickhouseodbcw.dylib"
+    end
+    lib.install_symlink "#{lib}/libclickhouseodbc.dylib" => "libclickhouseodbc.so"
+    lib.install_symlink "#{lib}/libclickhouseodbcw.dylib" => "libclickhouseodbcw.so"
+  end
+
+  test do
+    assert_match "/usr/lib/libSystem.B.dylib", shell_output("otool -L #{lib}/libclickhouseodbc.so")
```

Comment:
We need a test that exercises the some of the functionality of the app. Possibly a short example that compiles and links against the library.

Reply:
best testing is querying database via isql or iodbctest tools but they requires config file in home dir  ~/.odbc and running server.
Short example will test only linkage.

Reply:
Linkage is good

Reply:
done via ctest (test named load-ut)

Reply:
We need a piece of code that compiles to test the linkage against the library, not that the library links against system libraries.

Reply:
Something like what is done in https://github.com/Homebrew/homebrew-core/blob/master/Formula/libogg.rb

Reply:
inkage against the library tested inside ctest -V:
https://github.com/yandex/clickhouse-odbc/blob/master/driver/ut/CMakeLists.txt#L45
with
https://github.com/yandex/clickhouse-odbc/blob/master/driver/ut/load.cpp